dace/frontend/ml/__init__.py

```diff
@@ -0,0 +1,18 @@
+# Copyright 2019-2025 ETH Zurich and the DaCe authors. All rights reserved.
+
+try:
+    from .torch import DaceModule
+except ImportError:
+    DaceModule = None
+
+try:
+    from .onnx import ONNXModel
+except ImportError:
+    ONNXModel = None
+
+try:
+    from .tensorflow import TFSession
+except ImportError:
+    TFSession = None
```

Comment:
I don't like importing a deprecated frontend in an `__init__` file. Please remove this and others can import the submodule directly